modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -0,0 +1,36 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="creating-a-new-client-in-dex_{context}"]
+= Creating a new client in Dex
+
+Dex is used to delegate authentication to external identity providers like GitHub, SAML and others. To configure the SSO of Argo CD update the Argo CD CR with Dex link:https://dexidp.io/docs/connectors/[connector] settings.
```

Comment:
Please remove this line. We don't support Dex, we just support the Dex OpenShift connector.